helpers/GSGetter.ts

```diff
@@ -58,12 +66,18 @@ export class GCGetter {
                 break;
 
             case (Command.lgout):
-
-                // const mesg = modify.getCreator().startMessage().setSender(context.getSender()).setRoom(context.getRoom());
-                const logresponse = `https://www.google.com/accounts/Logout?continue=${redirect}`;
+                const logresponse = `https://www.google.com/accounts/Logout?continue=https://appengine.google.com/_ah/logout?continue=${redirect}`;
```

Comment:
```suggestion
                const logresponse = `https://www.google.com/accounts/Logout?continue=${redirect}`;
```

Reply:
The `redirect` variable will retrieve the site url as entered in the app settings, as for the rest of the url `https://www.google.com/accounts/Logout?continue=https://appengine.google.com/_ah/logout?continue=`, this is provided by google to log a user out and remains the same for all the users. just the site url which changes is retrieved dynamically.

Reply:
Sounds confusing 👀 Do you have a reference page?

Reply:
Yes sure, it is described in one of the answers here  -
 https://stackoverflow.com/questions/12909332/how-to-logout-of-an-application-where-i-used-oauth2-to-login-with-google